src/pages/ResearcherProfile.js

```diff
@@ -140,6 +141,20 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
         this.researcherFieldsValidation();
       }
     });
+    //clear out state field if the user selects
```

Comment:
before this code block was here when you would chose another country the state box would be disabled but still show the state value that was in it before